app/scripts/modules/netflix/ci/ci.dataSource.js

```diff
@@ -7,7 +7,7 @@ let angular = require('angular');
 module.exports = angular
   .module('spinnaker.netflix.ci.dataSource', [
     APPLICATION_DATA_SOURCE_REGISTRY,
-    require('./build.read.service'),
+    require('./services/build.read.service.js'),
```

Comment:
danged IDE - it must have added that when I moved the original service. Thanks for catching.